threescale_api/utils.py

```diff
@@ -49,12 +49,9 @@ def __init__(self, app, endpoint: str = "sandbox_endpoint",
         self._endpoint = endpoint
         if session is None:
             session = requests.Session()
-            retry = Retry(total=4, status_forcelist=(503,), raise_on_status=False,
-                          respect_retry_after_header=False)
+            self.retry_for_session(session)
 
-            adapter = HTTPAdapter(max_retries=retry)
-            session.mount("https://", adapter)
-            session.mount("http://", adapter)
+        if app.authobj is not None:
```

Comment:
this shouldn't be necessary anymore as we discussed that authobj can't be None unless it is correct value == authobj always returns right object or raises error

Reply:
Fixed